pages/portfolios/index.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+	<main>
+		<h1>
+			Portfolios.
```

Comment:
Formatting is off, should not have line breaks here. But I'll add linting and formatting with prettier to avoid that

Reply:
Formatting fixed